tests/local_mode/test_custom_doc_attr.py

```diff
@@ -0,0 +1,17 @@
+import syft as sy
+import torch
+import syfertext
+
+hook = sy.TorchHook(torch)
+me = hook.local_worker
+
+nlp = syfertext.load("en_core_web_lg", owner=me)
+
+def test_add_custom_attr_doc():
+    """Test adding custom attribute to Doc objects"""
+    
+    doc = nlp("Joey doesnt share food")
+    doc.set_attribute(name = 'my_custom_tag', value = 'tag')
+    
+    # check custom tag has been added
+    assert (hasattr(doc._,'my_custom_tag') and doc._.my_custom_tag == 'tag')
```

Comment:
can you please some more tests for:

- setting an existing attribute name (in this case the expected behaviour would be just to assign the new value to the existing attribute)

- attribute name is an empty string, or a string with a space in it (you can handle this with an assert statement in the set_attribute method. Later we will create our custom error class)

Reply:
Should I add more tests in the same function or create different function for each test?